core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3847,6 +3847,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_CACHE_EVICTOR_LRU_NONDETERMINISTIC_NUMOFCANDIDATE =
```

Comment:
Mentioned previously, let's not introduce such a flag for tthe number of candidate.

Instead, we can introduce `PropertyKey.USER_CLIENT_CACHE_EVICTOR_NONDETERMINISTIC`  as a boolean flag. Once this flag is `true` and `alluxio.user.client.cache.evictor.class` is LRUCacheEvictor, in `alluxio.client.file.cache.evictor.CacheEvictor#create` (the factory method of CacheEvictor, we can return this new NondeterministicLRUCacheEvictor`